### ISSUE TITLE
mdpow-equilibrium generate template broken

### ISSUE THREAD

**Issue by ianmkenney** [Becksteinlab/MDPOW#74]
Using the `mdpow-equilibrium --get-template` script does not return template.

```
mdpow       : INFO     MDPOW 0.6.1-dev starting.
mdpow       : INFO     Copyright (c) 2010-2016 Ian Kenney, Bogdan Iorga, and Oliver Beckstein
mdpow       : INFO     Released under the GNU Public Licence, version 3.
mdpow       : INFO     For bug reports and help: https://github.com/Becksteinlab/MDPOW/issues
mdpow.config: INFO     Using the bundled force fields from GMXLIB='/nfs/homes/ikenney/Library/python/MDPOW/mdpow/top'.
INFO:mdpow.config:Using the bundled force fields from GMXLIB='/nfs/homes/ikenney/Library/python/MDPOW/mdpow/top'.
mdpow.config: INFO     If required, override this behaviour by setting the environment variable GMXLIB yourself.
INFO:mdpow.config:If required, override this behaviour by setting the environment variable GMXLIB yourself.
mdpow       : CRITICAL A run input file is required See --help.
CRITICAL:mdpow:A run input file is required See --help.
```

**Comment by orbeckst** [comment]
How about we make it a separate script and get rid of `--get-template` — that's confusing anyway.

**Comment by orbeckst** [comment]
@ianmkenney can you have a look at this – this is the remaining main thing that needs to be fixed for the 0.6.1 release.
